.github/actions/deploy-centos/action.yml

```diff
@@ -145,9 +145,9 @@ runs:
 
         if [[ "$CI_DEPLOY_PLATFORM" == "linux-x86_64" ]] && [[ "$CI_DEPLOY_MODULE" == "spinnaker" ]]; then
           echo Installing Spinnaker amd64
-          python3 -m gdown.cli https://drive.google.com/uc?id=1zr9k4x-NI3OkVMLCccSBY4VtR-16iAQG
-          tar -xzvf spinnaker-2.4.0.143-Ubuntu18.04-amd64-pkg.tar.gz
-          find -name *.deb | while read f; do ar -xv $f; tar --totals -xvf data.tar.xz -C /; done;
+          python3 -m gdown.cli https://drive.google.com/uc?id=1ILDEZk0SuOoh4_cJExpZJCPXCqfxUkAi
+          tar -xzvf spinnaker-3.0.0.118-amd64-pkg.tar.gz
+          find -name *.deb | while read f; do ar -xv $f; tar -I zstd --totals -xvf data.tar.zst -C /; done;
```

Comment:
The files available from https://flir.app.boxcn.net/v/SpinnakerSDK/folder/133154514756 differ, why?

Reply:
That's very odd. I didn't change anything before uploading strait to Drive. I'll upload fresh and change the file ID.

Reply:
It looks like what changed is they reverted switching to zstd from xz, so let's also remove the changes related to that:
```
x - debian-binary
x - control.tar.xz
x - data.tar.xz
tar (child): data.tar.zst: Cannot open: No such file or directory
tar (child): Error is not recoverable: exiting now
tar: Child returned status 2
tar: Error is not recoverable: exiting now
```

Reply:
Ah, yep that would make sense. Haha.